src/jsi/test/testlib.cpp

```diff
@@ -721,8 +721,12 @@ TEST_P(JSITest, HostFunctionTest) {
           .getString(rt)
           .utf8(rt),
       "A cat was called with std::function::target");
+
+  // Disabling these tests for V8 because we'd incur unnecessary cost to implement the functionality
```

Comment:
>Disabling [](start = 5, length = 9)

So, is this planned to be implemented in the future? If not, should these tests be removed?

Reply:
I'm trying to keep the diff as small as possible, since this code is supposed to be shared with other engines' implementations.

---
In reply to: [492189122](https://github.com/microsoft/v8-jsi/pull/28#discussion_r492189122) [](ancestors = 492189122)